src/Console/Exporters/FluentExporter.php

```diff
@@ -51,6 +55,29 @@ public function exportClassMetadata(ClassMetadataInfo $metadata)
         return implode("\n", $lines);
     }
 
+    public function export()
+    {
+        if (!is_dir($this->_outputDir)) {
+            mkdir($this->_outputDir, 0775, true);
+        }
+
+        foreach ($this->_metadata as $metadata) {
+            // In case output is returned, write it to a file, skip otherwise
+            if ($output = $this->exportClassMetadata($metadata)) {
+                $path = $this->_outputDir.'/'.str_replace('\\', '.', $this->_mappingClassName).$this->_extension;
```

Comment:
I use `App\Infrastructure\Mappings\{entity_name}Mapping.php` in my projects. Not a fan of suffixes for domain classes, but I keep using them in non-domain space.